### PR TITLE
changes

### PR DESCRIPTION
**Pull request by mosesfawole**

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        </ul>
+      </div>
+    </header>
+    <div class="about">
+      <div class="head"><h4>About Us</h4></div>
+      <p>
+        Lorem, ipsum dolor sit amet consectetur adipisicing elit. Eum ipsam,
+        beatae quis aspernatur, soluta, praesentium earum ad magni asperiores
+        rerum vero veritatis repellat officia dolorum harum doloremque fugit.
+        Eaque, facere.
```
Comment (ogeobubu) [inline about.html:46]: [factservices-43a43727c756d53119e306eccf7a03588d6c2650.zip](https://github.com/ogeobubu/factservices/files/5477173/factservices-43a43727c756d53119e306eccf7a03588d6c2650.zip)

**Review by ogeobubu — CHANGES_REQUESTED** [review]

**Review by mosesfawole — COMMENTED** [review]
...

File: about.html (hunk 1 above)
Comment (ogeobubu) [reply about.html:46]: You've seen it right?

**Comment by mosesfawole** [comment]
i can't merge the pull request, its displaying this *Only those with write access to this repository can merge pull requests.*

**Comment by ogeobubu** [comment]
I just merged now

**Pull request merged**